cordova-plugin-outline/apple/src/OutlinePlugin.swift

```diff
@@ -295,4 +299,59 @@ class OutlinePlugin: CDVPlugin {
     }
     return callbackId
   }
+
+  // Migrates local storage files from UIWebView to WKWebView.
+  private func migrateLocalStorage() {
```

Comment:
This change is risky: if things go wrong, the clients would lose their list of servers.
I wonder how we can mitigate possible issues. I want to avoid the Electron localStorage issue that we saw before.

What if the app closes or crashes in the middle of the migration?

We should release this to beta testers first.

Reply:
Agreed. I'll think of possible things that could go wrong.

Reply:
I'm wondering if we can make the migration somewhat atomic, and avoid a corrupt state.
Either it fully succeeds or it doesn't.
I think you could accomplish that by creating a temporary directory, copy all the files into it, then rename it into the destination. You need to consider what other files are in `LocalStorage/`

Reply:
Done in 08be642. I implemented atomic file copy through the [Data class](https://developer.apple.com/documentation/foundation/nsdata/writingoptions/1411764-atomic).